### ISSUE TITLE
Promote the WCAG 2.2 testimonial video

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4448]
## What
Once published, promote the WCAG 2.2 video with the community.

## Why
To raise awareness and generate conversation.

## Who needs to work on this
Kelly

## Who needs to review this
N/A

## Done when
- [ ] the video has been published
- [ ] we have promoted the video at the next monthly call
- [ ] we have considered sharing via the mailing list
- [ ] we have considered sharing on social media
- [ ] we have considered sharing on Slack (xGov & GDS)
- [ ] we have considered sharing with SLT

**Comment by emma1carter** [comment]
No longer needed... will close
